release/models/bgp/openconfig-bgp-neighbor.yang

```diff
@@ -30,7 +30,14 @@ submodule openconfig-bgp-neighbor {
     "This sub-module contains groupings that are specific to the
     neighbor context of the OpenConfig BGP module.";
 
-  oc-ext:openconfig-version "8.0.0";
+
+  oc-ext:openconfig-version "9.0.0";
```

Comment:
Does this need to be a major revision change? If it's just a new leaf addition then it can be `8.1.0` I think.

Reply:
Updated. Thanks.